lib/rspec/rails/view_rendering.rb

```diff
@@ -6,6 +6,8 @@ module Rails
     module ViewRendering
       extend ActiveSupport::Concern
 
+      # @!attribute [r]
```

Comment:
I assume this is just here to declare the attribute is read-only, right?  Should we consider changing `attr_accessor` to `attr_reader` to actually make it read only?  And/or we could do `private :controller=` to make the writer private.